File: src/main/java/com/claudio_personagens/bookingcharactersapi/domain/CharacterGroup.java
package com.claudio_personagens.bookingcharactersapi.domain;

import com.claudio_personagens.bookingcharactersapi.base.AbstractBaseEntity;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.Entity;
import javax.persistence.ManyToMany;
import javax.persistence.OneToMany;
import java.util.Set;

@Data
@Entity
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class CharacterGroup extends AbstractBaseEntity {

    private float price;

    @OneToMany
    private Set<Character> characters;
}

File: src/main/java/com/claudio_personagens/bookingcharactersapi/domain/CharacterGroup.kt
package com.claudio_personagens.bookingcharactersapi.domain

import com.claudio_personagens.bookingcharactersapi.base.AbstractBaseEntity
import javax.persistence.Entity
import javax.persistence.OneToMany

@Entity
data class CharacterGroup(
    val price : Float = 0f,

    @OneToMany
    val characters: Set<Character>?
) : AbstractBaseEntity()